### DISCUSSION TITLE
Unable to Find Specific Word in Files with Telescope Plugin err:[telescope] ...\telescope\pickers.lua:488: in function <...telescope\pickers.lua:450>

### DISCUSSION THREAD

**Discussion by TwelveNight** [LazyVim/LazyVim#898] [Q&A] [answered]
Description:

I'm encountering an issue with the Telescope plugin in NeoVim. I'm trying to use Telescope to search for a specific word in files, but it's unable to find any matching results. Here are the specifics of the problem:

1. I'm using Linux as my operating system and NeoVim version 0.9.0.

2. I have successfully installed and configured the Telescope plugin, and other Telescope features are working fine.

3. I try to use the command `:Telescope find_string` to search for the specific word in files,but this command not find.

4. For example, I'm trying to search for the word "example" in the current working directory,The error message I received is as follows:
```shell
[telescope] ...\telescope\pickers.lua:488: in function <...telescope\pickers.lua:450>
```
5. I have confirmed that the word "example" does exist in the files.

I do not change my default config. I just add some plugins like tabnine.nvim，treesitter.nvim ...

Please assist me in resolving this issue and enabling Telescope to correctly find the specific word in files. Let me know if you need any additional information. Thank you!

**Comment by dpetka2001** [comment]
Do you mean maybe `grep_string` instead of `find_string`? Lazyvim already binds this command to `<leader>sw`. Have you tried that?

**Reply by TwelveNight** [reply]
I try sG st sw, all the same err

**Reply by dpetka2001** [reply]
Could you try re-install Telescope? Maybe something broken in your config? What do you mean you installed Telescope? Lazyvim comes by default with Telescope installed. Do you use Lazyvim or something else?

**Reply by folke** [reply]
This should all just work otb indeed. Definitely not a LazyVim issue. If any, you'd have to check with Telescope.

**Reply by TwelveNight** [reply]
> Could you try re-install Telescope? Maybe something broken in your config? What do you mean you installed Telescope? Lazyvim comes by default with Telescope installed. Do you use Lazyvim or something else?

I use Lazyvim install it, I also re-install.

**Reply by dpetka2001** [reply]
If you use Lazyvim you don't need to install it. It already comes pre-installed. Plz try to start fresh with a new Lazyvim starter configuration. Something must be broken with your config and it's really hard to figure out, since we don't have access to your configuration.

**Reply by TwelveNight** [reply]
> If you use Lazyvim you don't need to install it. It already comes pre-installed. Plz try to start fresh with a new Lazyvim starter configuration. Something must be broken with your config and it's really hard to figure out, since we don't have access to your configuration.

Did you mean I should use other template rather than the starter?
I use this now
```
git clone https://github.com/LazyVim/starter ~/.config/nvim
```
Do you have any recommendations?

**Reply by dpetka2001** [reply]
Yes, I meant that. Just follow the steps and install it again. Don't forget to remove/backup as per the [docs](https://www.lazyvim.org/installation)
```shell
mv ~/.local/share/nvim ~/.local/share/nvim.bak
mv ~/.local/state/nvim ~/.local/state/nvim.bak
mv ~/.cache/nvim ~/.cache/nvim.bak
```
and install again from scratch. Does the same thing happen? Because it shouldn't happen, unless you broke something with your configuration.

**Reply by TwelveNight** [reply]
I'm sorry that I aready try it,but nothing change
I even installed it with my cloud server, but the error message was the same

**Reply by dpetka2001** [reply]
Did you remove the old configuration files?

**Reply by TwelveNight** [reply]
> Did you remove the old configuration files?

If you mean bakeup,I just try what as you say

**Reply by dpetka2001** [reply]
Also you might want to try installing the latest stable 9.1 Neovim. I'm using the nightly version and also installed the starter template from scratch without any configuration and it worked without any problems.

**Reply by TwelveNight** [reply]
> Also you might want to try installing the latest stable 9.1 Neovim. I'm using the nightly version and also installed the starter template from scratch without any configuration and it worked without any problems.

What version of neovim you are using?

**Reply by TwelveNight** [reply]
> > Also you might want to try installing the latest stable 9.1 Neovim. I'm using the nightly version and also installed the starter template from scratch without any configuration and it worked without any problems.
> 
> What version of neovim you are using?

my version is the official stable version 0.9.1

**Reply by TwelveNight** [reply]
I also try use linuxbrew install Neovim,the version is 0.9.0 
```shell
NVIM v0.9.0
Build type: Release
LuaJIT 2.1.0-beta3

     system vimrc file: "$VIM/sysinit.vim"
         $VIM default value: "
/home/linuxbrew/.linuxbrew/Cellar/neovim/0.9.0/share/nvim"

Run :checkhealth for more info
```

**Reply by dpetka2001** [reply]
No idea really. The starter template without any configuration should work out of the box to do what you want. Something in your configuration is causing the problem. You might also want to open an issue at Telescope repo, since they are more likely to help you if it's a specific Telescope problem.

**Reply by TwelveNight** [reply]
Thanks for your suggestion, I will open an issue with Telescope

**Reply by TwelveNight** [reply]
![2](https://github.com/LazyVim/LazyVim/assets/113519679/a36b1cdf-7c80-4363-9829-560e54edc837)
Is this the problem?

**Reply by dpetka2001** [reply]
It might be. Telescope uses either `ripgrep` or `fd` for grep functions. You should install them.

**Reply by TwelveNight** [reply, marked as answer]
> It might be. Telescope uses either `ripgrep` or `fd` for grep functions. You should install them.

Yes!Is this problem.
